acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/SignerConfigurationBuilder.java

```diff
@@ -32,7 +34,20 @@ public SignerConfigurationBuilder withWebSocketPort(final int port) {
     return this;
   }
 
+  public SignerConfigurationBuilder withHashicorpVaultPort(final int port) {
+    hashicorpVaultPort = port;
+    return this;
+  }
+
+  public SignerConfigurationBuilder withIpAddress(final String address) {
```

Comment:
Is this the hashicorp vault ip address? If so be good prefix this with hashicorp to make that clear

Reply:
done